src/Native/System.Net.Security.Native/pal_gssapi.cpp

```diff
@@ -102,9 +102,11 @@ extern "C" uint32_t NetSecurityNative_DisplayStatus(uint32_t* minorStatus,
     assert(outBuffer != nullptr);
 
     int statusType = isGssMechCode ? GSS_C_MECH_CODE : GSS_C_GSS_CODE;
+    uint32_t messageContent;
     GssBuffer gssBuffer {.length = 0, .value = nullptr};
     uint32_t majorStatus =
-        gss_display_status(minorStatus, statusValue, statusType, GSS_C_NO_OID, nullptr, &gssBuffer);
+        gss_display_status(minorStatus, statusValue, statusType, GSS_C_NO_OID, &messageContent, &gssBuffer);
+    (void)messageContent; // unused
```

Comment:
It is a flag that is populated by the framework.  The name of the parameter in the header file is ```message_content```. So I used the same name with our naming conventions.

> Interesting... was this necessary? I'm surprised the compiler warned in this case. Is that because it's attributed to be an out value on OS X or something?

About compiler failing, it did not. But then, on my OSX box, it was not failing anyway. I am not sure what is the difference between CI machine and mine is. I can give a try without this statement.